### PR TITLE
mdl_project_tex_coords

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.03%) on `us`.

🚀 This PR matches 1 function (+0.03%) on `ique`.

File: src/a5dd0_len_114e0.c [hunk 1]
```diff
@@ -5690,87 +5689,70 @@ Gfx* mdl_get_copied_gfx(s32 copyIndex) {
...
     Gfx* dlist;
+    s32 cmd;
+    Vtx* tempVert;
+
+    s8 zero; // TODO needed to match
 
     listIndex = get_model_list_index_from_tree_index(modelID & 0xFFFF);
     model = get_model_from_list_index(listIndex);
     dlist = model->modelNode->displayData->displayList;
+    zero = 0;
```
Comment (nanaian) [inline src/a5dd0_len_114e0.c:5737]: We're certain `zero` needs to be given a value after these func calls and not at declaration?

File: src/de740_len_23f0.c [hunk 2]
```diff
@@ -272,6 +272,12 @@ void create_shading_palette(Matrix4f mtx, s32 uls, s32 ult, s32 lrs, s32 lrt, s3
     }
 }
 
+extern int ENVIRONMENT;
+extern int SHADE;
+extern int COMBINED;
+extern int COMBINED_ALPHA;
+extern int TEXEL0_ALPHA;
```
Comment (nanaian) [inline src/de740_len_23f0.c:279]: What are these? They don't appear to be used

Comment (ethteck) [reply src/de740_len_23f0.c:279]: they're to help with the permuter while this function is WIP

**Pull request merged**